CONTRIBUTING.md

```diff
@@ -7,3 +7,78 @@ Before submitting a PR / pulishing a release make sure to:
 * Update and commit changes to README.md with: `npm run builder:build`
 
 Thanks for helping out!
```

Comment:
Can you add:

```
## Pull Requests
```

as a similar heading above the section above that begins with `Before submitting`?